x/bounty/client/cli/tx.go

```diff
@@ -0,0 +1,132 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/shentufoundation/shentu/v2/x/bounty/types"
+)
+
+// NewTxCmd returns the transaction commands for the certification module.
+func NewTxCmd() *cobra.Command {
+	bountyTxCmds := &cobra.Command{
+		Use:   "bounty",
+		Short: "Bounty transactions subcommands",
+	}
+
+	bountyTxCmds.AddCommand(
+		NewCreateProgramCmd(),
+	)
+
+	return bountyTxCmds
+}
+
+func NewCreateProgramCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-program",
+		Short: "create new program initialized with an initial deposit",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			creatorAddr := clientCtx.GetFromAddress()
+			if err != nil {
+				return err
+			}
+
+			desc, _ := cmd.Flags().GetString(FlagDesc)
+
+			encKeyFile, _ := cmd.Flags().GetString(FlagEncKeyFile)
+			var encKey cryptotypes.PubKey
+			if encKeyFile == "" {
```

Comment:
if encKeyFile is empty, the module will help generate new pubkey and prikey.  This means bounty module will help generate different prikey.  The SaveKeys function should be to save different file.

Reply:
https://github.com/shentufoundation/shentu/issues/536